config/locales/en.yml

```diff
@@ -2838,12 +2840,14 @@ en:
   text_work_packages_destroy_confirmation: "Are you sure you want to delete the selected work package(s)?"
   text_work_packages_ref_in_commit_messages: "Referencing and fixing work packages in commit messages"
   text_journal_added: "%{label} %{value} added"
-  text_journal_changed_html: "%{label} changed from %{old} %{linebreak}<strong>to</strong> %{new}"
+  text_journal_changed_html: "%{label} changed from %{old} %{linebreak}to %{new}"
```

Comment:
`text_journal_changed_html` can be removed from the code to only use `text_journal_changed_plain` instead.

I would like to rename it `text_journal_changed` but that means additional work for the translators, with the risk of having `text_journal_changed` not translated, which is too bad because `text_journal_changed_plain` is already translated and correct.

Also using `text_journal_changed_plain` instead of `text_journal_changed_html` allows us to benefit from the right translation without waiting for the translation of `text_journal_changed_html` to be updated.

Again, removing code reduces future maintenance efforts.